libraries/matchers.rb

```diff
@@ -60,7 +60,7 @@
     oneview_unmanaged_device:           %i[add remove add_if_missing],
     oneview_uplink_set:                 standard_actions,
     oneview_user:                       standard_actions,
-    oneview_volume:                     standard_actions + %i[create_snapshot delete_snapshot],
+    oneview_volume:                     standard_actions + %i[add create_from_snapshot create_snapshot delete_snapshot],
```

Comment:
Why are we using `add` now? With this, we'd have both `create` and `add` for this resource, and I don't see the particular need for this. Especially since in API500 we really do create the volumes anyway.
Any reasoning behind it?

Reply:
In API 500, the endpoints are different as well as the payload. For this reason, I believe that the two must be separated, because the two actions are different. In the add action, I don't create a volume, I add an existing volume, and the documentation does not cite how I check if that volume is an external volume, and that I can understand that it is an addition and not a creation. Perhaps the way to differentiate one or the other is by passing the device_volume_name property. What do you think?